handsontable/test/helpers/common.js

```diff
@@ -559,6 +559,28 @@ export function triggerTabNavigationFromBottom(hotInstance = hot()) {
   $(hotInstance.rootWrapperElement).find('.htFocusCatcher').last().focus();
 }
 
+/**
+ * Emulates the browser's TAB navigation out of the Handsontable (to element above).
+ */
+export function triggerTabNavigationToAbove() {
+  // Trigger Shift+Tab to navigate to element above
+  // We'll use the global keyDownUp function that should be available in test environment
+  if (typeof keyDownUp === 'function') {
```

Comment:
The `keyDownUp` helper is always available in the test workspace. Why then is the `triggerTabNavigationToAbove` helper needed? 🤔